docker/install.sh

```diff
@@ -9,10 +10,17 @@ if [ -e "jenkins_env.sh" ]; then
 	source "jenkins_env.sh"
 fi
 
+# Assumes docker
+docker pull mysql/mysql-server:5.7
+docker run -p 3306:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.7
+
 # catch sbt issues separately
 sbt update && echo "sbt updated successfully."
 
-sbt -J-Xms4g -J-Xmx4g test
+# Assumes docker-machine
+sbt -J-Xms4g -J-Xmx4g test -Ddockermysql.ip=`docker-machine ip default`
```

Comment:
This is actually going to execute within the docker build so we need to be able to run docker in our docker. But I don't really like how this is turning out. Is running the tests in the docker build process the right thing? I feel like this is becoming too heavy weight.